nixos/modules/services/misc/moonraker.nix

```diff
@@ -0,0 +1,141 @@
+{ config, lib, pkgs, ... }:
+with lib;
+let
+  cfg = config.services.moonraker;
+  format = pkgs.formats.ini {
+    listToValue = l: "\n" + builtins.concatStringsSep "\n" (map (elem: "  ${elem}") l);
+    mkKeyValue = generators.mkKeyValueDefault {} ":";
+  };
+in {
+  options = {
+    services.moonraker = {
+      enable = mkEnableOption "Moonraker, an API web server for Klipper";
+
+      package = mkOption {
+        type = types.package;
+        default = pkgs.moonraker;
+        description = "The Moonraker package.";
+      };
```

Comment:
The package setting is mostly for places where there can realistically be multiple co-existing packages in `nixpkgs` that serve the same function (e.g. multiple versions of samba).

I don't think this is the case for Moonraker, we only have one package for it. If users want they can still override it with an overlay.